### DISCUSSION TITLE
notify_for_audit偶尔发送通知失败

### DISCUSSION THREAD

**Discussion by Tan-Pchan** [hhyo/Archery#2722] [General]
报错信息具体查看
![image](https://github.com/hhyo/Archery/assets/68987741/ef1a968b-ad18-4dbe-b9dc-2d4924459c1c)

**Comment by LeoQuote** [comment]
这个可能比较难排查, 我给你几个排查的思路看一下:
1. 升级到最新的 master 代码
2. 接一个 [sentry](https://docs.sentry.io/platforms/python/integrations/django/) 看一下堆栈
3. 使用相同的工单或者事件, 尝试在本地复现, 打印更多的日志
4. 更改代码打印更多日志, 尝试看到具体是哪里的传参有问题.

**Comment by Tan-Pchan** [comment]
上面的问题暂未定位到。
遇到另一个问题：钉钉推送消息有推动到队列里。但没收到钉钉消息，从archery.log也没看到有相关日志，后台也没有失败记录
之前版本未遇到，升级到v1.10.0后才遇到
![image](https://github.com/user-attachments/assets/b8a93f51-30ec-402a-bdcb-f24ecfc8dbb7)
![image](https://github.com/user-attachments/assets/c023731d-8179-4766-8808-5bc2cc535d15)

**Reply by LeoQuote** [reply]
还请按上述方案排查试试，仅凭你提供的信息我无法复现bug协助你排查
